packages/warriorjs-core/src/Space.test.js

```diff
@@ -198,6 +202,10 @@ describe('Space', () => {
         expect(space.isEnemy()).toBe(true);
```

Comment:
We should remove this test. Also, [the group](https://github.com/olistic/warriorjs/pull/101/files#diff-16985e5c8289f152dae95fcb370cfab4R200) should be called `hostile` instead of `enemy` now.